### DISCUSSION TITLE
Twilio Segment: side and top navigation shifted pixel

### DISCUSSION THREAD

**Discussion by karoltonik** [twilio-labs/paste#3372] [Help: Figma] [answered]
I noticed that both the side navigation and top navigation have their width and height shifted by 1 px due to the separator that is enabled in the component area. This happens because the "strokes: included" option is turned on in the auto layout options. Changing this option to "strokes: excluded" would solve the issue. The protruding pixel could cause several problems, such as uneven spacing, odd distances contrary to the 8pt grid system guidelines, and misalignment relative to the grid.

[Figma link](https://www.figma.com/file/oeUbF1qpn2FUsmXwiTsHmN/Segment-App%3A-Paste-Migration?type=design&node-id=1113%3A401320&mode=design&t=bAE3PKDNod0y9JWh-1)

**Comment by serifluous** [comment, marked as answer]
Hey @karoltonik ! Good catch on the Sidebar and Topbar—A fix for this will be published soon in the Paste Components library.

I'll reply again here when it's published.

**Reply by serifluous** [reply]
Just published the fix, thanks!
